config/node1.yml

```diff
@@ -0,0 +1,94 @@
+- type: fetch_cluster
+  request:
+    fetch_cluster_request:
+      linearizable: true
+  response:
+    fetch_cluster_response:
+      type:
+        - leader_response
+        - leader_response
+        - leader_response
+- type: propose
+  request:
+    propose_request:
+      seq_num: 1
+  response:
+    propose_response:
+      type: default_response
```

Comment:
We’d better to define the content of the request and response directly in the configuration file, rather than pre-defining possible responses with strings. This is inflexible, and if a new test requires a response type that is not predefined, we need to modify the code instead of the configuration file